runtime/polkadot/src/lib.rs

```diff
@@ -2142,6 +2142,10 @@ sp_api::impl_runtime_apis! {
 				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef780d41e5e16056765bc8461851072c9d7").to_vec().into(),
 				// Treasury Account
 				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef7b99d880ec681799c0cf30e8886371da95ecffd7b6c0f78751baa9d281e0bfa3a6d6f646c70792f74727372790000000000000000000000000000000000000000").to_vec().into(),
```

Comment:
all of these can be removed i think, since they are now whitelisted directly in the pallet

Reply:
Should I replace it only on the polkadot runtime?

Westend for example whitelists two storage keys from the `Dmp` pallet, `DownwardMessageQueues` and `DownwardMessageQueueHeads`. If I were to use `#[pallet::whitelist_storage]` on these items, suddenly every other runtime would also have them whitelisted.

Is that desirable? Or is there a reason why we would want to whitelist them in westend and not the other runtimes?

Reply:
most pallets in the polkadot repo apply only to the polkadot runtime, but in general, if these storage items are read every block on Polkadot, it will be read every block on every chain. So yes, it is likely you should update all the pallets to use this system.